sysutils/diskus/Portfile

```diff
@@ -0,0 +1,86 @@
+    rand                             0.5.5  e464cd887e869cddcae8792a4ee31d23c7edd516700695608f5b98c67ee0131c \
+    rand_core                        0.2.2  1961a422c4d189dfb50ffa9320bf1f2a9bd54ecb92792fb9477f99a1045f3372 \
+    rand_core                        0.3.0  0905b6b7079ec73b314d4c748701f6931eb79fd97c668caa3f1899b22b32c6db \
+    rayon                            1.0.3  373814f27745b2686b350dd261bfd24576a6fb0e2c5919b3a2b6005f820b0473 \
+    rayon-core                       1.4.1  b055d1e92aba6877574d8fe604a63c8b5df60f60e5982bf7ccbb1338ea527356 \
+    redox_syscall                   0.1.42  cf8fb82a4d1c9b28f1c26c574a5b541f5ffb4315f6c9a791fa47b6a04438fe93 \
+    redox_termios                    0.1.1  7e891cfe48e9100a70a3b6eb652fef28920c117d366339687bd5576160db0f76 \
+    rustc_version                    0.2.3  138e3e0acb6c9fb258b19b67cb8abd63c00679d2851805ea151465464fe9030a \
+    scopeguard                       0.3.3  94258f53601af11e6a49f722422f6e3425c52b06245a5cf9bc09908b174f5e27 \
+    semver                           0.9.0  1d7eb9ef2c18661902cc47e535f9bc51b78acd254da71d375c2f6720d9a40403 \
+    semver-parser                    0.7.0  388a1df253eca08550bef6c72392cfe7c30914bf41df5269b68cbd6ff8f570a3 \
+    smallvec                         0.6.6  622df2d454c29a4d89b30dc3b27b42d7d90d6b9e587dbf8f67652eb7514da484 \
+    stable_deref_trait               1.1.1  dba1a27d3efae4351c8051072d619e3ade2820635c3958d826bfea39d59b54c8 \
+    strsim                           0.7.0  bb4f380125926a99e52bc279241539c018323fab05ad6368b56f93d9369ff550 \
+    term_size                        0.3.1  9e5b9a66db815dcfd2da92db471106457082577c3c278d4138ab3e3b4e189327 \
+    termion                          1.5.1  689a3bdfaab439fd92bc87df5c4c78417d3cbe537487274e9b0b2dce76e92096 \
+    textwrap                        0.10.0  307686869c93e71f94da64286f9a9524c0f308a9e1c87a583de8e9c9039ad3f6 \
+    unicode-width                    0.1.5  882386231c45df4700b275c7ff55b6f3698780a650026380e72dabe76fa46526 \
+    unreachable                      1.0.0  382810877fe448991dfc7f0dd6e3ae5d58088fd0ea5e35189655f84e6814fa56 \
+    vec_map                          0.8.1  05c78687fb1a80548ae3250346c3db86a80a7cdd77bda190189f2d0a0987c81a \
+    void                             1.0.2  6a02e4885ed3bc0f2de90ea6dd45ebcbb66dacffe03547fadbb0eeae2770887d \
+    winapi                           0.2.8  167dc9d6949a9b857f3451275e911c3f44255842c1f7a76f33c55103a909087a \
+    winapi                           0.3.6  92c1eb33641e276cfa214a0522acad57be5c56b10cb348b3c5117db75f3ac4b0 \
+    winapi-build                     0.1.1  2d315eee3b34aca4797b2da6b13ed88266e6d612562a0c46390af8299fc699bc \
+    winapi-i686-pc-windows-gnu       0.4.0  ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6 \
+    winapi-x86_64-pc-windows-gnu     0.4.0  712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f
```

Comment:
I am not familiar with cargo, but are all these hardcoded versions and hash codes not avoidable ?

Reply:
@cjones051073 I'm not really familiar with `cargo` either, so I couldn't tell you for sure. I got that big ugly list by using [`cargo2port.tcl`](https://github.com/macports/macports-contrib/blob/master/cargo2port/cargo2port.tcl) as described in [`cargo_fetch-1.0.tcl`.](https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/cargo_fetch-1.0.tcl) From what I can see in the `ripgrep` and `fd` ports, these big lists might be a common theme for `cargo` ports.

Reply:
I wasn't following closely, but I'm pretty sure that this is sadly in fact the case.

The only thing that we could do (but is outside of scope of this PR) is to provide checksums in a separate file. Something in the spirit of `crossgcc-1.0.tcl`.